integration/configure_windows_test.go

```diff
@@ -1,5 +1,5 @@
-// +build !linux
-//
+// +build configure configure,!linux
```

Comment:
is this right? or should it be 
```suggestion
// +build configure,!linux
```

Reply:
`spaces are OR and commas are AND`

So this reads "include me when specifying configure, or when specifying configure and not linux"

Reply:
TIL 🙌🏽